hs_core/views/utils.py

```diff
@@ -59,6 +61,26 @@ def upload_from_irods(username, password, host, port, zone, irods_fnames, res_fi
         fname = os.path.basename(ifname.rstrip(os.sep))
         res_files.append(UploadedFile(file=tmpFile, name=fname, size=size))
 
+# run the update script on hyrax server via ssh session for netCDF resources on demand
+# when private netCDF resources are made public so that all links of data services
```

Comment:
@pkdash Not exactly, this function is only executed every time when the netcdf resource ```public``` attribute is set to True, i.e., when a private resource is made public. Since the script will be run as a cron job on hyrax server to clean things up when public resource is made private or deleted, there is no need to run it on-demand for those cases. Your question made me realize I did not ensure this in the code logic. I will make changes tomorrow morning and let you know once I am done so that you can give a final review. Thanks for catching this.